src/App/Fossa/FossaAPIV1.hs

```diff
@@ -250,6 +250,7 @@ renderIssueType = \case
 
 data Issue = Issue
   { issueId :: Int
+  , issuePriorityString :: Maybe Text -- we only use this field for `fossa test --json`
```

Comment:
It feels weird to do a roundtrip serialization for json output, but I think this is correct